apps/web/assets/js/components/Opportunity.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const translated_difficulty = (level) => {
+  var term;
+
+  switch (level) {
+    case 1: 
+      term = "Starter";
+      break;
+    case 5:
+      term = "Intermediate";
+      break;
+    case 9:
+      term = "Advanced";
+      break;
+  }
+
+  return term;
+}
+
+export default class Opportunity extends React.Component {
+  render() {
```

Comment:
If you get tired of dealing with the verbosity of `this.props.thing.1.2.3` you can always do some destructuring at the top the render function, like so:

```js
const { project, url } = this.props.data;
```

Then you can just reference `project.tags` instead of `this.props.data.project.tags` for the the rest of the render function.

Reply:
Wow, that's awesome.  TIL!  Thanks @cafreeman